ewlegal.cls

```diff
@@ -0,0 +1,84 @@
+%% TODO Stock beginning text, copyrights and project info and all that
+
+%% Work consists of the files engwales-legal.cls and accompanying engwales-*.sty
+
+
+\NeedsTeXFormat{LaTeX2e}
+\ProvidesClass{ewlegal}[2021-02-21 Legal Templates for England and Wales]
+
+%% Uses article as a base
+\LoadClass[12pt,a4paper]{article}% TODO Perhaps use KOMA-Script?
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Options
+%%%%%
+
+%% Easy option management
+\RequirePackage{kvoptions}
+
+%% Select the template used
+% TODO Be able to nicely handle multiple templates being selected
+
+\DeclareBoolOption[false]{pleading}
+
+\DeclareBoolOption[false]{opinion}
+
+
+%% Extras
+
+
+
+%% Process options given by the user
+\ProcessKeyvalOptions*
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Packages
+%%%%%
+
+%% Generic Packages
+
+%% Easy formatting of section titles
+\RequirePackage[explicit]{titlesec}
+
+%% Allows for easy use of logic
+\RequirePackage{etoolbox}
+
+%% Fancy headers and footers
+\RequirePackage{fancyhdr}
+
+%% Spaced out boxes
+\RequirePackage{soul}
+% Doesn't seem to be reccommended anymore due to issues with luatex
+
+% What does this do?
+\RequirePackage[normalem]{ulem}
+
+%% Expand the feature set of typeset lists
+\RequirePackage{enumitem}
+
+%% Handle margins
+\RequirePackage[margin=2.5cm]{geometry}
+% Should the end user always have to declare this? Or should we try for
+% reasonable defaults?
```

Comment:
i used 2.5 cm because the example Particulars of Claim doc used 2.5 cm. but i was curious if that's required too. @passamo9 is a margin of 2.5 cm required or just your preference?